modules/local/hisat2/align/main.nf

```diff
@@ -0,0 +1,68 @@
+process HISAT2_ALIGN {
+    tag "$meta.id"
+    label 'process_high'
+
+    // WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-a97e90b3b802d1da3d6958e0867610c718cb5eb1:2cdf6bf1e92acbeb9b2834b1c58754167173a410-0' :
+        'biocontainers/mulled-v2-a97e90b3b802d1da3d6958e0867610c718cb5eb1:2cdf6bf1e92acbeb9b2834b1c58754167173a410-0' }"
+
+    input:
+    tuple val(meta), path(reads) //reads may contain multiple pairs of fastq: [fastq1_1, fastq2_1, fastq1_2, fastq2_2], it can handle multiple pairs of fastq, but the read group information can not be added properly. When run with single pair of fastq, the read group info is added properly
+    tuple val(meta2), path(index)
+    tuple val(meta3), path(splicesites)
+
+    output:
+    tuple val(meta), path("*.hisat2_Aligned.bam")    , emit: bam
+    tuple val(meta), path("*_summary.txt")           , emit: summary
+    tuple val(meta), path("*fastq.gz"), optional:true, emit: fastq
```

Comment:
Is this for unmapped reads?

I'm not sure we're producing unmapped reads:
https://daehwankimlab.github.io/hisat2/manual/#:~:text=in%20the%20input.-,%2D%2Dun%2Dconc,-%3Cpath%3E%2C